plugin/coqsimpleio.mlg.cppo

```diff
@@ -144,12 +144,11 @@ let define_and_run c env evd =
   let packages =
     match !extra_pkg with
     | [] -> ""
-    | [x] -> "-pkg " ^ x
-    | _ -> failwith "SimpleIO ERROR: implement more than one linked package"
+    | x -> "-pkgs " ^ (String.concat "," x)
   in
   let exec_command =
     match !dune_file with
-    | None -> "cd " ^ dir ^ "; ocamlbuild " ^ packages ^ " -lib unix -cflags \"-w -3\" " ^ Filename.basename execn ^ " -quiet > build.log 2> build.err"
```

Comment:
Does stuff using `Unix` still build without that flag?

Reply:
No longer builds with that flag, OCaml complains seeing two Unix libraries, thus removing the flag.